test/xpu_api/utilities/pairs/pairs.pair/ctor1_g.pass.cpp

```diff
@@ -0,0 +1,83 @@
+// -*- C++ -*-
```

Comment:
This test seems valueless and can be removed.

Reply:
Done